meddlr/data/build.py

```diff
@@ -243,13 +243,16 @@ def build_recon_train_loader(cfg, dataset_type=None):
     )
     return train_loader
 
-
 def build_recon_val_loader(
     cfg,
     dataset_name,
     as_test: bool = False,
     add_noise: bool = False,
     add_motion: bool = False,
+    angle: float = 0,
+    translation: float = 0,
+    nshots: int = 0,
+    trajectory: str = "blocked", 
```

Comment:
we want to avoid taking in too many arguments - consider passing in a `data_transform`

Reply:
If I pass in just a data_transform as a parameter (for instance like MotionDataTransform), how would I pass in the parameters for angle, translation, nshots, and trajectory into the MotionDataTransform? Thanks, Deepro.